generic3g/OuterMetaComponent.F90

```diff
@@ -315,25 +337,115 @@ recursive subroutine initialize(this, importState, exportState, clock, unusable,
 
       integer :: status, userRC
       type(ChildComponent), pointer :: child
+      type(OuterMetaComponent), pointer :: child_meta
       type(ChildComponentMapIterator) :: iter
 
+      associate (phase => get_phase_index(this%phases_map%of(ESMF_METHOD_INITIALIZE), phase_name='GENERIC_INIT_GRID', rc=status))
+        if (status == _SUCCESS) then
+           call ESMF_GridCompInitialize(this%user_gridcomp, importState=importState, exportState=exportState, &
+                clock=clock, userRC=userRC, phase=phase, _RC)
```

Comment:
Could we have the userRC and _RC always to be the last two arguments of the ESMF calls

Reply:
NP.  Please use the "add a suggestion" (+/- box) to make such requests in the future.  Makes it easier to manage.
```suggestion
                clock=clock, phase=phase, userRC=userRC, _RC)
```